javascript/javascript3/week3/shopping.js

```diff
@@ -0,0 +1,128 @@
+// sometimes it showing 403 failed loading error
```

Comment:
Also, this part is a separate task, so could be in separate files named something related to the task :)